nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/groovy/org/apache/nifi/fingerprint/FingerprintFactoryGroovyTest.groovy

```diff
@@ -84,7 +85,7 @@ class FingerprintFactoryGroovyTest extends GroovyTestCase {
         logger.info("Read initial flow: ${initialFlowXML[0..<100]}...")
 
         // Create the FingerprintFactory with collaborators
-        FingerprintFactory fingerprintFactory = new FingerprintFactory(mockEncryptor, extensionManager)
+        FingerprintFactory fingerprintFactory = new FingerprintFactory(mockEncryptor, extensionManager, SecureHasherFactory.getSecureHasher())
```

Comment:
This test also has access to `nifi.properties`, so recommend using the configured `algorithm` property.